classes/data-stores/ActionScheduler_DBStore.php

```diff
@@ -1015,7 +1017,18 @@ public function log_execution( $action_id ) {
 
 		$sql = "UPDATE {$wpdb->actionscheduler_actions} SET attempts = attempts+1, status=%s, last_attempt_gmt = %s, last_attempt_local = %s WHERE action_id = %d";
 		$sql = $wpdb->prepare( $sql, self::STATUS_RUNNING, current_time( 'mysql', true ), current_time( 'mysql' ), $action_id ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
-		$wpdb->query( $sql ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+
+		// phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+		if ( ! $wpdb->query( $sql ) ) {
```

Comment:
More of a personal style thing, I suppose, but I usually try to avoid executing logic within a conditional's expression, as it makes it harder for me to find where something happened when scanning the code. 🤷

Reply:
Thanks, @coreymckrill: more than happy to update it—to confirm, is this what you'd prefer to see?

```php
$status_updated = $wpdb->query( $sql );

if ( ! $status_updated ) {
```

Reply:
Yeah, I think that's easier to follow 👍